Makefile

```diff
@@ -57,28 +57,47 @@ RBAC_LIST = rbac.authorization.k8s.io_v1_clusterrole_platform-operators-manager-
 	rbac.authorization.k8s.io_v1_clusterrolebinding_platform-operators-manager-rolebinding.yaml \
 	rbac.authorization.k8s.io_v1_clusterrolebinding_platform-operators-proxy-rolebinding.yaml \
 	rbac.authorization.k8s.io_v1_role_platform-operators-leader-election-role.yaml \
-	rbac.authorization.k8s.io_v1_rolebinding_platform-operators-leader-election-rolebinding.yaml
+	rbac.authorization.k8s.io_v1_rolebinding_platform-operators-leader-election-rolebinding.yaml \
+	rbac.authorization.k8s.io_v1_clusterrole_platform-operators-rukpak-bundle-reader.yaml \
+	rbac.authorization.k8s.io_v1_clusterrole_platform-operators-rukpak-bundle-uploader.yaml \
+	rbac.authorization.k8s.io_v1_clusterrole_platform-operators-rukpak-core-admin.yaml \
+	rbac.authorization.k8s.io_v1_clusterrolebinding_platform-operators-rukpak-core-admin.yaml
 
 # Generate manifests e.g. CRD, RBAC etc.
 .PHONY: manifests
 manifests: generate kustomize
 	$(KUSTOMIZE) build config/default -o $(TMP_DIR)/
-	$(KUSTOMIZE) build config/rukpak -o $(TMP_DIR)/rukpak.yaml
 	ls $(TMP_DIR)
 
-	@# now rename/join the output files into the files we expect
-	mv $(TMP_DIR)/rukpak.yaml manifests/0000_50_cluster-platform-operator-manager_00-rukpak.yaml
-	mv $(TMP_DIR)/apiextensions.k8s.io_v1_customresourcedefinition_platformoperators.platform.openshift.io.yaml manifests/0000_50_cluster-platform-operator-manager_00-platformoperator.crd.yaml
-	mv $(TMP_DIR)/v1_namespace_openshift-platform-operators-system.yaml manifests/0000_50_cluster-platform-operator-manager_00-namespace.yaml
-	mv $(TMP_DIR)/v1_serviceaccount_platform-operators-controller-manager.yaml manifests/0000_50_cluster-platform-operator-manager_01-serviceaccount.yaml
-	mv $(TMP_DIR)/v1_service_platform-operators-controller-manager-metrics-service.yaml manifests/0000_50_cluster-platform-operator-manager_02-metricsservice.yaml
-	mv $(TMP_DIR)/apps_v1_deployment_platform-operators-controller-manager.yaml manifests/0000_50_cluster-platform-operator-manager_06-deployment.yaml
+	@# Move all of the rukpak manifests into the manifests folder
+	mv $(TMP_DIR)/apiextensions.k8s.io_v1_customresourcedefinition_bundledeployments.core.rukpak.io.yaml manifests/0000_60_cluster-platform-operator-manager_00-rukpak-bundledeployments.crd.yaml
```

Comment:
should we include `mv` as part of the variable? Something like $MV_TMP_DIR=`mv $(TMP_DIR)`?

Reply:
That feels cleaner than it is now, yeah. We should consider potentially making a commit into `operator-sdk` that updates that there as well as this is the default.

Reply:
Quick note: this repository wasn't bootstrapped using operator-sdk, and the manifests target here is custom/hacky logic built on top of kubebuilder.

Reply:
Ahh, gotcha gotcha. Appreciate that context then. I added the update so our hacky logic becomes a bit cleaner in that case.